client-src/elements/form-field-specs.js

```diff
@@ -1134,6 +1141,51 @@ export const ALL_FIELDS = {
     help_text: '',
   },
 
+  'enterprise_policies': {
+    type: 'input',
+    attrs: MULTI_STRING_FIELD_ATTRS,
+    required: false,
+    label: 'Enterprise policies',
+    help_text: html`
+      List of policies that control the feature, if any.`,
+  },
+
+  'rollout_milestone': {
+    type: 'input',
+    attrs: MILESTONE_NUMBER_FILED_ATTRS,
+    required: true,
+    label: 'Rollout milestone',
+    help_text: html`
+      Milestone in which rollout for this feature starts.`,
+  },
+
+  'rollout_platforms': {
+    type: 'multiselect',
+    choices: PLATFORM_CATEGORIES,
+    required: true,
+    label: 'Rollout platforms',
+    help_text: html`
+      Platforms for which rollout for this feature occurs in the selected milestone.`,
+  },
+
+  'rollout_details': {
+    type: 'textarea',
+    attrs: {rows: 4},
+    required: false,
+    label: 'Rollout details',
+    help_text: html`
+      Explain what specifically is changing in the selected milestone for the 
+      selected platforms. Include any controls admins have to test it 
+      (e.g. flags) and control it (e.g. an enterprise policy). Write in the 
+      present tense.`,
+  },
+
+  'breaking_change': {
+    type: 'checkbox',
+    label: 'Breaking change',
+    initial: false,
+    help_text: 'This is a breaking change (customers/developers must take action)',
```

Comment:
I think that we can do better with this help text, but let's leave that for a follow-up PR.

Reply:
Ack.